docs/pywbemlistener/generaloptions.rst

```diff
@@ -0,0 +1,78 @@
+.. _`pywbemlistener general options`:
+
+Pywbemlistener general options
+------------------------------
+
+General options are those that are specified directly after the ``pywbemlistener``
+command, e.g. ``--output-format simple`` in the following example:
+
+.. code-block:: text
+
+    pywbemlistener --output-format simple list
+
+Most of the general options can be specified with any of the sub-commands of the
+``pywbemlistener`` command, but there are some general options that are used
+only on some sub-commands, and are ignored on the other sub-commands.
+
+
+.. index:: single: pywbemlistener logging
+
+.. _`pywbemlistener logging`:
+
+Logging
+"""""""
+
+Logging is performed only by the ``pywbemlistener run`` command. The
+``pywbemlistener start`` command passes its logging settings on to the
+``pywbemlistener run`` command it starts as a background process. All other
+``pywbemlistener`` commands ignore the logging settings.
+
+Logging is enabled by specifying the ``-l DIR`` or ``--logdir DIR`` general
+option, or by setting the ``PYWBEMLISTENER_LOGDIR`` environment variable to
+``DIR``. The general option has precedence over the environment variable.
+``DIR`` is the path name of the directory where the log files will
+reside in.
+
+The log files are named ``pywbemlistener_NAME.log`` where ``NAME`` is the
```

Comment:
Add there is a log file created for each indication listener initiated (or something like that).

Reply:
DONE: Added "There is a log file created for each listener that is running."